src/main.cc

```diff
@@ -509,6 +520,11 @@ main(int argc, char** argv)
     triton::server::signal_exit_cv_.wait_for(lock, wait_timeout);
   }
 
+  // Stop the HTTP[, gRPC, and metrics] endpoints, and update exit timeout.
+  uint32_t exit_timeout_secs = g_triton_params.exit_timeout_secs_;
+  StopEndpoints(&exit_timeout_secs);
+  TRITONSERVER_ServerSetExitTimeout(server_ptr, exit_timeout_secs);
```

Comment:
Is error checking required?

Reply:
The error is a part of the API, in case it is needed for the future, but it does not currently produce an error, since it is only assigning a parameter value to a class object.

If there is an error in setting core exit timeout, nothing can be done here but either push forward with the shutdown (with a possibly incorrect timeout value for the core) or force exit immediately. If the timeout value is incorrect on the core, it is visible on the server log during exit and an additional error message I think is redundant.